.github/workflows/auto-reqs-update-pr.yml

```diff
@@ -17,6 +18,17 @@ jobs:
           # find other PRs from that repo
           other_prs=$(curl -s ${renku_repo}/pulls | jq ".[] | select((.head.ref | test(\"auto-update/${pr_repo}\")) and (.head.ref != \"${GITHUB_REF:11}\")) | .number")
           for pr in $other_prs ; do curl -s -H "Authorization: token $GITHUB_TOKEN" -X PATCH -d '{"state": "closed"}' ${renku_repo}/pulls/${pr} ; done
+
+          pr_body="This is an automated pull request.\n\n/deploy"
```

Comment:
Can we add the bash code about the body of the automated pr in a separate step? It has nothing to do with closing PRs so it is really weird to see it there. We can also name this step appropriately for what it does to avoid confusion.